src/core/mormot.core.interfaces.pas

```diff
@@ -983,6 +983,7 @@   TInterfaceResolverInjected = class(TInterfaceResolver)
       overload; virtual;
     /// check if a given interface can be resolved, from its RTTI
     function Implements(aInterface: PRttiInfo): boolean; override;
+    procedure DeleteResolver(aResolver: TInterfaceResolver);

```

Comment:
please add a /// comment (with 3 /) before the method for documentation consistency

Reply:
Done